governance/reviewers-maintainers.md

```diff
@@ -1,40 +1,16 @@
-Unlike other Special Interest Groups (SIGs), the O3DE Documentation and Community SIG maintains two reviewer roles requiring SIG approval: *Documentation reviewer* and *technical reviewer*. Learn how to become either, and file a [nomination issue](https://github.com/o3de/sig-docs-community/issues/new?assignees=&labels=&template=reviewer-maintainer-nomination.md&title=SIG+Reviewer%2FMaintainer+Nomination) to become an official reviewer or maintainer!
+## Documentation and Community reviewer roles
 
-For more on the Reviewer and Maintainer roles, read our [Community Membership](https://github.com/o3de/community/blob/main/community-membership.md) information. 
+The Documentation and Community SIG maintains the following reviewer teams:
 
-----
-<!-- CUT ABOVE THIS LINE BEFORE POSTING -->
-### Nomination Guidelines
+* [**Documentation Reviewer**](https://github.com/orgs/o3de/teams/docs-reviewers) - Responsible for reviewing documentation content to `o3de.org`, or conducting reviews of certain `o3de` in-product documentation elements.
+* **Community Reviewer** - Responsible for reviewing community materials, including contributor guides and blogs.
 
-#### Documentation reviewer
+We also have [SIG maintainers](https://github.com/o3de/o3de/discussions). Documentation and Community maintainers are reviewers who have been elevated from the documentation or community reviewer pools to take on additional responsibility, and serve as trusted sources of guidance.
 
-_Documentation reviewers_ review for conformance with the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/), appropriateness of content, and other subjective factors to keep documentation content aligned. 
+### Technical reviewer roles
 
-To become a documentation reviewer, you must have:
+The Documentation and Community SIG also maintains a documentation technical reviewer team for the `o3de.org` website and its related infrastructure. Documentation technical reviewers are responsible for representing their SIG during documentation review, and ensuring technical accuracy of content.
```

Comment:
Tried to make this clearer about the tech reviewer role and removed the link for now. If we automate the system of review assignment, it might make more sense to have each SIG maintain its own independent reviewer group rather than contribute to a global pool.

Reply:
Great point! I agree with this. I want to be cautious about automating review assignment for other SIG's technical review teams because they may want to handle that differently.